src/audio/codec_adapter/codec/cadence.c

```diff
@@ -238,8 +238,8 @@ static int cadence_codec_apply_config(struct comp_dev *dev)
 		API_CALL(cd, XA_API_CMD_SET_CONFIG_PARAM, param->id,
 			 param->data, ret);
 		if (ret != LIB_NO_ERROR) {
-			comp_err(dev, "cadence_codec_apply_config() error %x: failed to apply parameter %d value %d",
-				 ret, param->id, *(int32_t *)param->data);
+			comp_warn(dev, "cadence_codec_apply_config(): applied parameter %d value %d with return code: %#x",
```

Comment:
we don't know if its fatal at this point, shouldnt we split this based on the following check?

Reply:
good point. fixed now